### PR TITLE
Changed how duplicate attributes are displayed when adding manual events in the sketch API client

### PR DESCRIPTION
**Pull request by warrinot**
Fixing issue #1733.
This is my first open-source issue, please tell me if i did something wrong.

### CODE REVIEW

File: api_client/python/timesketch_api_client/sketch.py [hunk 1]
```diff
@@ -1522,8 +1522,17 @@ def add_event(
...
             'tag': tags
         }
-        if any(x in attributes for x in form_data):
-            raise ValueError('Attributes cannot overwrite values already set.')
+
+        duplicate_attributes = list()
+
+        for key in attributes.keys():
+            if key in form_data.keys():
+                duplicate_attributes.append(key)
```
Comment (kiddinn) [inline api_client/python/timesketch_api_client/sketch.py:1530]: for this you could also just have done

```python
duplicate_attributes = [k for k in attributes if k in form_data]
```

that is do that in a single line using list comprehension

**Review by kiddinn — CHANGES_REQUESTED** [review]
thank you for the patch, made a minor suggestion

File: api_client/python/timesketch_api_client/sketch.py [hunk 2]
```diff
@@ -1522,8 +1522,17 @@ def add_event(
...
+
+        if duplicate_attributes:
+            duplicates = ', '.join(duplicate_attributes)
+            raise ValueError(
+                f'Following attributes cannot overwrite values already set: {duplicates}')
```
Comment (kiddinn) [inline api_client/python/timesketch_api_client/sketch.py:1535]: there is a linter issue here...

**Review by kiddinn — COMMENTED** [review]
forgot one thing

**Comment by kiddinn** [comment]
Why did you close this PR?

**Comment by warrinot** [comment]
I was not sure if you could see my changes and messed up with commits a bit.

**Pull request closed without merging**